integrations/facebook-pixel/test/index.test.js

```diff
@@ -147,6 +148,12 @@ describe('Facebook Pixel', function() {
         );
       });
 
+      it('should call dataProcessingOptions if limitedDataUse is enabled', function() {
```

Comment:
maybe overkill -- but also add test for when it is disabled?

Reply:
Good call. I'll add one!